db/seeds.rb

```diff
@@ -5,3 +5,4 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+Message.create(greeting: "Hello pal! Have a nice day!")
```

Comment:
- [ ] Instead of creating just one greeting, you should have created array of object of messages for the random picking to be possible as required for the project.
Sample :point_down: 

```
   Message.create([
    { greeting: "Hello pal! Have a nice day 1!" },
    { greeting: "Hello pal! Have a nice day 2!" },
    { greeting: "Hello pal! Have a nice day! 3" },
    { greeting: "Hello pal! Have a nice day! 4" },
    { greeting: "Hello pal! Have a nice day! 5" },
])
``` 

### Project Requirement
![greeting](https://user-images.githubusercontent.com/26597432/232043118-0c527ebc-b56b-4169-abdd-337a90b2d31d.png)

I was able to generate random greetings with the above snippet. Kindly consider adding this to your seed file.

Reply:
done :heavy_check_mark:  Thanks for the review @Qoosim